src/Configuration/Config.AzureTableStorage/src/AzureTableStorageConfigurationExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.Extensions.Configuration.AzureTableStorage;
+using System;
+
+namespace Microsoft.Extensions.Configuration
+{
+    /// <summary>
+    /// Configuration extensions for AzureTableStorage configuration provider
+    /// </summary>
+    public static class AzureTableStorageConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds AzureTableStorage configuration to the configuration builder
+        /// </summary>
+        /// <param name="configurationBuilder"></param>
```

Comment:
Nit: missing description. See for example https://github.com/aspnet/Extensions/blob/7fd2ab9f058570ce241a19cb12bdf8d13151a861/src/Configuration/Config.CommandLine/src/CommandLineConfigurationExtensions.cs#L19